src/framework/lib/database.ts

```diff
@@ -102,6 +104,52 @@ export class Database {
             }
         });
     }
+
+    /**
+     * Returns an array of migration files to run.
+     */
+    public static getMigrationFiles(currentVersion ?: string) : MigrationFile[] {
+        let migrationsPath = pub('migrations');
+        let currentVersionInt = (currentVersion) ? this.getVersionInt(currentVersion) : 0;
+
+        let fileNames = fs.readdirSync(migrationsPath).filter(name => !name.equals('init.sql')).sort((a, b) => {
+            let aParts = a.replace('.sql', '').split('.');
+            let bParts = b.replace('.sql', '').split('.');
+
+            while (aParts.length < 3) aParts.push('0');
+            while (bParts.length < 3) bParts.push('0');
+
+            let aId = (1000000 * parseInt(aParts[0])) + (1000 * parseInt(aParts[1])) + parseInt(aParts[2]);
+            let bId = (1000000 * parseInt(bParts[0])) + (1000 * parseInt(bParts[1])) + parseInt(bParts[2]);
+
+            return aId - bId;
```

Comment:
You wrote a function that does most of this already. Consider simplifying this filter to 
`return this.getVersionInt(a) - this.getVersionInt(b);`